docs/components/gantry/multi-axis.md

```diff
@@ -202,8 +202,8 @@ Edit and fill in the attributes as applicable.
 {{% /tab %}}
 {{% /tabs %}}
 
-The following attributes are available for `multiaxis` gantries:
+The following attributes are available for `multi-axis` gantries:
 
 | Attribute | Type | Inclusion | Description |
 | --------- | ---- | --------- | ----------- |
-| `subaxes_list` | array | **Required** | An array of the `name` of each of the sub-axes, the [one-axis](/components/gantry/oneaxis/) gantries that make up the multi-axis gantry. |
+| `subaxes_list` | array | **Required** | An array of the `name` of each of the sub-axes, the [`single-axis`](/components/gantry/single-axis/) gantries that make up the `multi-axis` gantry. |
```

Comment:
[opt] outside of diff, but if you want to make this sentence less awk while you're at it, I don't have a good idea but maybe you do!

Reply:
Hmm, explored a bit but don't have one either at the moment. Might circle back later.